native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarBinaryExpression.scala

```diff
@@ -130,8 +130,9 @@ object ColumnarBinaryExpression {
         new ColumnarFromUnixTime(left, right)
       case d: DateSub =>
         new ColumnarDateSub(left, right)
-      case g: GetJsonObject =>
-        new ColumnarGetJsonObject(left, right, g)
+      //TODO(): the current impl has poor perf
+      // case g: GetJsonObject =>
+      //   new ColumnarGetJsonObject(left, right, g)
       case instr: StringInstr =>
```

Comment:
@PHILO-HE note this patch disabled json parsing temporary

Reply:
I see. Thanks for the reminding.